meinberlin/apps/bplan/management/commands/bplan_get_location.py

```diff
@@ -1,25 +1,20 @@
 from django.core.management.base import BaseCommand
 
 from meinberlin.apps.bplan.models import Bplan
-from meinberlin.apps.bplan.tasks import get_bplan_api_pk_to_a4_admin_district_dict
-from meinberlin.apps.bplan.tasks import get_bplan_point_and_district_pk
+from meinberlin.apps.bplan.tasks import get_bplan_point
 
 
 class Command(BaseCommand):
     help = (
         "Uses the Bplan identifier to get the coordinates "
-        "and the district from API of bplan-prod.liqd.net"
```

Comment:
you removed the destrict because you are adding it in the custom save() of the bplan model?

Reply:
yes, exactly, we don't need to fetch it from the bplan map anymore